docs/Homebrew-Governance.md

```diff
@@ -16,131 +17,132 @@
+6. Any member may refer any financial questions, AGM questions or code of conduct violations to the PLC. All technical matters must instead be referred to the Project Leader and technical disputes to the TSC. Members will make a good faith effort to resolve any disputes with compromise prior to referral to the PLC, Project Leader or TSC.
 
 ## 6. Project Leader
 
 1. The Project Leader will represent Homebrew publicly, manage all day-to-day technical decisions, and resolve disputes related to the operation of Homebrew between maintainers, members, other contributors, and users.
 
-2. The Project Leader will be elected annually by Homebrew members in a [Schulze Condorcet method](https://en.wikipedia.org/wiki/Schulze_method) (aka 'beatpath') election. The PLC will nominate at least one candidate for Project Leader. Any member may nominate a candidate, or self-nominate. Nominations must be announced to the membership three weeks before the AGM.
+2. The Project Leader will be elected every two years by Homebrew members in a [Schulze Condorcet method](https://en.wikipedia.org/wiki/Schulze_method) (aka 'beatpath') election. The PLC will nominate at least one candidate for Project Leader. Any member may nominate a candidate, or self-nominate. Nominations must be announced to the membership three weeks before the AGM.
 
 3. Any vacancy of the Project Leader will be filled by appointment of the PLC.
 
-4. The Project Leader's seat on the PLC is non-voting, unless a tie-breaker vote is required.
-
-5. A technical decision of the Project Leader may be overruled by an ordinary resolution of the TSC.
+4. A technical decision of the Project Leader may be overruled by an ordinary resolution of the TSC.
 
-6. A non-technical decision of the Project Leader may be overruled by an ordinary resolution of the PLC.
+5. A non-technical decision of the Project Leader may be overruled by an ordinary resolution of the PLC.
 
-7. The Project Leader may be removed from the position by a special resolution of the membership.
+6. The Project Leader must only be removed from the position by a special resolution of the membership.
 
-8. The Project Leader must be included in all PLC communications with or about Open Collective and in all communications related to joint responsibilities.
+7. The Project Leader must be included in all PLC communications with or about Open Collective and in all communications related to joint responsibilities.
 
-9. The Project Leader must be a maintainer, not just a member.
+8. The Project Leader must be a maintainer, not just a member.
 
-10. The Project Leader will be an "Owner" of the GitHub organization, Slack, 1Password and any related resources.
+9. The Project Leader will be an "Owner" of the GitHub organization, Slack, 1Password and any related resources.
```

Comment:
... of the Project's GitHub organization, Slack workspace, 1Password vault (account?) and any other resources related to Project's operations.

Reply:
Owner is a specific type of user for an account in 1Password.